src/sentry/monitors/validators.py

```diff
@@ -148,7 +161,7 @@ def validate(self, attrs):
         return attrs
 
 
-class MonitorValidator(serializers.Serializer):
+class MonitorValidator(CamelSnakeSerializer):
```

Comment:
nice 💯

Reply:
annoyed I have to put this on the top level object but it won't work if it's only on the nested validator